src/ui/windows/TogglDesktop/TogglDesktop/utilities/Utils.cs

```diff
@@ -41,7 +41,9 @@ public static void LoadWindowLocation(Window mainWindow, EditViewPopup editPopup
         }
         else
         {
-            Toggl.Debug("Failed to retrieve window location and size");
+            mainWindow.Width = 330;

```

Comment:
The default height and width look good to me.

Do you think we should reset the top left of the window to zero too? So in worst case, the x y is out of screen, and it could reset to the origin position.